api/app.js

```diff
@@ -49,6 +50,21 @@ app.use(cookieParser());
 // application routes
 app.use('/', indexRouter);
 app.use(['/profile', '/profiles'], profileRouter);
+app.use(
+  [
+    '/workOrders?',
```

Comment:
Might just be ignorance on my part again, but why are you using an array of endpoints instead of just one root endpoint?